### DISCUSSION TITLE
Slider with custom marks

### DISCUSSION THREAD

**Discussion by petro-shkuratenyuk** [primefaces/community#711] [PrimeReact]
is it possible to implement slider with custom marks using PrimeReact same way it implemented here: <a href="https://mui.com/material-ui/react-slider/#custom-marks">material-ui custom-marks</a>?

**Comment by melloware** [comment]
See: https://github.com/primefaces/primereact/issues/4778

**Reply by petro-shkuratenyuk** [reply]
it's not really the same thing: 
 - what I was asking is slider with points, where there is no way to select position between the points
 - what you sugested is pure slider with labels on top (with no change to functionality)

**Reply by melloware** [reply]
You would have to request and enhancement ticket.
